data/src/main/java/com/linkedin/data/codec/AbstractJacksonDataCodec.java

```diff
@@ -57,10 +58,12 @@
    *
    * <a href="https://github.com/FasterXML/jackson-docs/wiki/Presentation:-Jackson-Performance">Jackson Performance</a>
    */
-  public static final JsonFactory JSON_FACTORY = new JsonFactory();
+  public static final JsonFactoryBuilder JSON_FACTORY_BUILDER = new JsonFactoryBuilder();
```

Comment:
Builder can be moved to static block

Reply:
this is just perference, right?

Reply:
this would save space also. we don't need to keep reference to it in codec.